okcurl/src/main/kotlin/okhttp3/curl/Main.kt

```diff
@@ -271,5 +283,24 @@ class Main : Runnable {
         })
       }
     }
+
+    private fun enableSslDebugging() {
+      sslLogger = Logger.getLogger("javax.net.ssl").apply {
+        level = Level.FINE
+        addHandler(ConsoleHandler().apply {
+          level = Level.FINE
+          formatter = object : SimpleFormatter() {
+            override fun format(record: LogRecord): String {
+              val parameters = record.parameters
+              if (parameters != null) {
+                return format("%s%n%s%n", record.message, record.parameters.first())
```

Comment:
Always one parameter?

Reply:
It's almost JSON as a single param.

See example here https://13821-5152285-gh.circle-artifacts.com/0/%7E/reports/okhttp/build/reports/tests/test/classes/okhttp3.CallKotlinTest.html